content/en/integrations/guide/source-code-integration.md

```diff
@@ -181,15 +181,29 @@ COPY .git ./.git
 {{< tabs >}}
 {{% tab "GitHub" %}}
 
-If you are a GitHub SaaS user, install Datadog's [GitHub integration][1] on the [GitHub integration tile][2] to link your telemetry with your source code. When specifying permissions on the integration tile, enable Datadog read permissions to **Contents**.
+Install Datadog's [GitHub integration][1] on the [GitHub integration tile][2] to link your telemetry with your source code. When specifying permissions on the integration tile, enable Datadog read permissions to **Contents**.
 
 By setting up the GitHub integration, you can see inline code snippets in **Error Tracking**. For more information, see [Inline Source Code](#inline-source-code).
 
 [1]: https://docs.datadoghq.com/integrations/github/
 [2]: https://app.datadoghq.com/integrations/github/
 {{% /tab %}}
+{{% tab "GitLab" %}}
+
+<div class="alert alert-warning">
+Self-hosted instances or private URLs are not supported.
+</div>
+
+If you are a GitLab SaaS user, no setup is required to link your telemetry with your source code.
```

Comment:
Customers will have code snippets out of the box, but they still need to configure `datadog-ci git-metadata upload` to link telemetry. Should we remove this section, and only have special treatment for GitLab on the `Inline source code` sectio below ?